src/slurm_plugin/clustermgtd.py

```diff
@@ -907,7 +908,69 @@ def _run_clustermgtd():
                 e,
             )
         # Manage cluster
-        cluster_manager.manage_cluster()
+        cluster_manager.manage_cluster(custom_metrics)
+        custom_metrics._has_received = True
+        sleep_remaining_loop_time(config.loop_time, start_time)
+
+
+class ClusterMetrics:
+    def __init__(self):
+        self._has_received = False
+        self._slurm_active_nodes = None
+        self._slurm_inactive_nodes = None
+        self._ec2_nodes = None
+        self._config = None
+
+    def get_config(self):
```

Comment:
instead of getter/setter let's use properties

Reply:
Done for config.